en/README.md

```diff
@@ -33,7 +33,7 @@ MAVLink was first released early 2009 by Lorenz Meier and has now a [significant
 
 ## Supported Languages {#supported_languages}
 
-The MAVLink project includes the [mavgen](getting_started/generate_source.md#mavgen) generator tool that can be used to create MAVLink APIs/libraries for a number of programming languages (these are supported by the project). 
+The MAVLink project includes the [mavgen](getting_started/generate_source.md#mavgen) GUI tool that can be used to create MAVLink APIs/libraries for a number of programming languages. 
```

Comment:
mavgen is a command line too.. 
Perhaps that should say:

> includes the [mavgen](getting_started/generate_source.md#mavgen) and [mavgenerate](getting_started/generate_source.md#mavgenerate) tools

Reply:
mavgen is part of pymavlink. Mavgenerate sits on top, but is in the mavlink repo. Nuanced I know.

Reply:
By your logic the text you have is even more wrong - mavgen is not part of the project and it is not a GUI tool :-). To be completely correct you'd have to change this to "The MAVLink project includes the [mavgenerate](getting_started/generate_source.md#mavgenate) GUI tool that ..."

But what I have suggested is better. Because the MAVLink project logically includes both mavgenerate and mavgen (if only via submodule) and that is what users will care about. So here I am referring to the concept of project, not the actual mavlink/mavlink repo.


1. As you have it the text is even more wrong. By this logic - mavgen is   way the text would be wrong - because mavgen is not in the MAVLink project or a GUI tool as indicated.

Reply:
Philosophically I’d agree, but the actual links are all for mavgenerate.py even when they say mavgen in this document. This is the simplest “correct” fix.

Reply:
The links will go to mavgen and mavgenerate - I'm just waiting on a PR to be reviewed. The "right thing" to do is pointing people to the tools they will use, not worrying about what repo they come in from. 

So lets do:

> includes the [mavgen and mavgenerate](getting_started/generate_source.md#mavgen) tools.